core/test/unit/utils/blog-icon-spec.js

```diff
@@ -135,13 +136,56 @@ describe('Blog Icon', function () {
         });
     });
 
-    describe.skip('getIconDimensions', function () {
-        it('[success] returns icon dimensions', function (done) {
-            done();
+    describe('getIconDimensions', function () {
+        it('[success] returns .ico dimensions', function (done) {
+            blogIcon.getIconDimensions(path.join(__dirname, '../../utils/fixtures/images/favicon.ico'))
+                .then(function (result) {
+                    should.exist(result);
+                    result.should.eql({
+                        width: 48,
+                        height:48
+                    });
+                    done();
+                }).catch(done);
+        });
+
+        it('[success] returns .png dimensions', function (done) {
+            blogIcon.getIconDimensions(path.join(__dirname, '../../utils/fixtures/images/favicon.png'))
+                .then(function (result) {
+                    should.exist(result);
+                    result.should.eql({
+                        width: 100,
+                        height:100
+                    });
+                    done();
+                }).catch(done);
+        });
+
+        it('[success] returns .ico dimensions for icon with multiple sizes', function (done) {
+            blogIcon.getIconDimensions(path.join(__dirname, '../../utils/fixtures/images/favicon_multi_sizes.ico'))
+                .then(function (result) {
+                    should.exist(result);
+                    result.should.eql({
+                        width: 64,
+                        height:64
+                    });
+                    done();
+                }).catch(done);
         });
 
         it('[failure] return error message', function (done) {
-            done();
+            var sizeOfStub = sandbox.stub();
+
+            sizeOfStub.throws({error: 'image-size could not find dimensions'});
+
+            blogIcon.__set__('sizeOf', sizeOfStub);
```

Comment:
I recommend resetting this change after it is made, otherwise if another test is added after here in future it would be affected by this test.